cartographer/mapping/trajectory_node.h

```diff
@@ -67,8 +66,6 @@ struct TrajectoryNodes {
   std::vector<mapping::TrajectoryNode> trajectory_nodes;
 };
 
-proto::Trajectory ToProto(const std::vector<TrajectoryNode>& nodes);
```

Comment:
@macmason Is it possible tocreate a PR for `cartographer_ros` in advance? I would like to see how you handled `ToProto` and `FromProto` in `cartographer_ros`. And if possible, I want to contribute on top of your changes.